### The data structure for battery information

# The battery level
float32 battery_level
